addons/website_blog/migrations/10.0.1.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+from odoo.tools import safe_eval
+
+
+def convert_blog_post_cover(env):
+    """Switch opacity value for posts, as now in v10 the meaning of that
+    property is for an upper solid frame, not for the image itself.
+    """
+    post_obj = env['blog.post'].with_context(active_test=False)
+    for post in post_obj.search([('cover_properties', '!=', False)]):
+        props = safe_eval(post.cover_properties)
```

Comment:
upstream uses JSON here, so I think you should do the same: https://github.com/OCA/OCB/blob/10.0/addons/website_blog/controllers/main.py#L166